examples/ostack.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
Is there a reason not to call this file openstack.py?

Reply:
Ahh, right. Import errors are fun. open_stack is fine, or maybe openstack_example.py. 

Whatever seems best to you.